index.js

```diff
@@ -2538,6 +2668,7 @@
     PositiveNumber: PositiveNumber,
     RegExp: RegExp_,
     RegexFlags: RegexFlags,
+    Strict: Strict,
     StrMap: fromUncheckedUnaryType(StrMap),
     String: String_,
```

Comment:
I've been using `Array#sort` to determine order:

```javascript
> ['Strict', 'StrMap', 'String'].sort()
['StrMap', 'Strict', 'String']
```

Please also move the definition of `Strict`.